app/src/main/java/in/testpress/testpress/ui/ForumActivity.java

```diff
@@ -216,6 +221,11 @@ protected void onCreate(final Bundle savedInstanceState) {
             setEmptyText(R.string.invalid_post, R.string.try_after_sometime,
                     R.drawable.ic_error_outline_black_18dp);
         }
+        DaoSession daoSession = TestpressApplication.getDaoSession();
+        InstituteSettingsDao instituteSettingsDao = daoSession.getInstituteSettingsDao();
+        instituteSettings = instituteSettingsDao.queryBuilder()
+                .where(InstituteSettingsDao.Properties.BaseUrl.eq(BASE_URL))
+                .list().get(0);
```

Comment:
```suggestion
        instituteSettings = TestpressApplication.getInstituteSettings();
```

It can be extracted as it is a common method which is being in many places